gatsby-node.js

```diff
@@ -142,9 +144,122 @@ async function createBlogPages({ graphql, actions, reporter }) {
   );
 }
 
+// Create Events Page
+async function createEventsPage({ graphql, actions }) {
+  const { createPage } = actions;
+
+  const result = await graphql(
+    `
+      query ($draftFilter: [Boolean]!, $eventRegex: String!) {
+        allMdx(
+          filter: {
+            fileAbsolutePath: { regex: $eventRegex }
+            fields: { draft: { in: $draftFilter } }
+          }
+        ) {
+          totalCount
+        }
+        allTypes: allMdx(filter: { fileAbsolutePath: { regex: $eventRegex } }) {
+          group(field: frontmatter___type) {
+            fieldValue
+          }
+        }
+        allRegions: allMdx(filter: { fileAbsolutePath: { regex: $eventRegex } }) {
+          group(field: frontmatter___region) {
+            fieldValue
+          }
+        }
+        allPosts: allMdx(
+          filter: {
+            fileAbsolutePath: { regex: $eventRegex }
+            fields: { draft: { in: $draftFilter }, isFeatured: { eq: false } }
+          }
+          sort: { fields: frontmatter___date, order: DESC }
+        ) {
+          nodes {
+            frontmatter {
+              date(formatString: "MMMM DD, YYYY")
+              region
+              place
+              type
+              title
+              ogSummary
+              externalUrl
+              ogImage {
+                childImageSharp {
+                  gatsbyImageData(width: 550)
+                }
+              }
+            }
+          }
+        }
+        featuredPost: allMdx(
+          filter: {
+            fileAbsolutePath: { regex: $eventRegex }
+            fields: { draft: { in: $draftFilter }, isFeatured: { eq: true } }
+          }
+          limit: 1
+        ) {
+          nodes {
+            frontmatter {
+              date(formatString: "MMMM DD, YYYY")
+              region
+              place
+              type
+              title
+              ogSummary
+              externalUrl
+              ogImage {
+                childImageSharp {
+                  gatsbyImageData(width: 735)
+                }
+              }
+            }
+          }
+        }
+      }
+    `,
+    { draftFilter: DRAFT_FILTER, eventRegex: EVENT_REGEX }
+  );
+
+  if (result.errors) throw new Error(result.errors);
+
+  const { totalCount } = result.data.allMdx;
+  const {
+    featuredPost: { nodes: featuredPost },
+    allPosts: { nodes: allEvents },
+    allTypes: { group: allTypes },
+    allRegions: { group: allRegions },
+  } = result.data;
+
+  function getFrontmatterData(items) {
+    return items.map((item) => ({ ...item.frontmatter }));
+  }
+
+  const postEvents = getFrontmatterData(allEvents);
+  const featuredEvent = featuredPost[0].frontmatter;
+  const types = allTypes.reduce((acc, type) => [...acc, type.fieldValue], []);
+  const regions = allRegions.reduce((acc, type) => [...acc, type.fieldValue], []);
+  const eventFilters = createEventFilters(types, regions);
+  const initialFilters = getInitialFilters(eventFilters);
```

Comment:
```suggestion
  const initialFilters = allFilters.reduce((acc, { label }) => {
    if (!acc[label]) {
      acc[label] = [];
    }
    return acc;
  }, {});
```

Reply:
IMHO, as it was not a reusable function, it is a shorter way to just get initialFilters here, not importing from utils.